Cargo.toml

```diff
@@ -32,4 +32,4 @@ hamcrest = "~0.1.5"
 default = ["netsim"]
 [target."cfg(target_os = \"linux\")".dependencies.netsim]
 optional = true
-version = "~0.2.2"
+version = "~0.2.3"
```

Comment:
Why are we updating the version in a non-version change PR ?

Reply:
It's a good practice, but in Crust we don't have a hard rule to keep commits atomic if compared to Routing team git conventions.

Reply:
This only a version change for the netsim dependency, not the crate as a whole.

Reply:
ah, such an attentive chap, ain't I. lol - sorry merging